compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEVectorization.cpp

```diff
@@ -77,6 +77,7 @@ void AMDAIEVectorizationPass::runOnOperation() {
     // dominate this use'. TODO(newling) follow-up on this.
     if (isa<linalg::FillOp>(op)) return;
 
+    if (isElementwise(cast<linalg::LinalgOp>(op))) return;
```

Comment:
I know this is originally written by me, but could you add some comments above to explain why we disable the elementwise op for vectorization?